queue_services/entity-digital-credentials/src/entity_digital_credentials/digital_credentials_processors/put_back_on.py

```diff
@@ -0,0 +1,31 @@
+# Copyright © 2023 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Processing put back on actions."""
+
+
+from legal_api.models import Business, DCRevocationReason
+
+from entity_digital_credentials.helpers import get_issued_digital_credentials, revoke_issued_digital_credential
+
+
+async def process(business: Business):
+    """Process put back on actions."""
+    issued_credentials = get_issued_digital_credentials(business=business)
+
+    if not (issued_credentials and len(issued_credentials)):
+        raise Exception('No issued credentials found.')
+
+    return revoke_issued_digital_credential(business=business,
+                                            issued_credential=issued_credentials[0],
+                                            reason=DCRevocationReason.PUT_BACK_ON)
```

Comment:
A put back can only occur when a company was put into a historical state via an administrative dissolution filing.  

My thinking is that when the administrative dissolution occurs, a revocation would have occurred for the business.

Do we need to do another revocation when a put back on occurs for this business?

Reply:
Good point. Probably a business (Linda) question, but my guess is no.

Reply:
^^ I confirmed with Linda...

1. SPs don't usually do PBOs
2. Do not revoke on PBO. The credential would have been revoked on a dissolution.
3. Users can re-issue a credential if they want. Don't do it automatically.

Akiff, you were on this meeting. Correct me if I'm wrong.

Reply:
I believe we confirmed that the current business logic is correct. Will mark this as resolved.